rules/symfony/src/Rector/MethodCall/FormTypeInstanceToClassConstRector.php

```diff
@@ -90,7 +91,7 @@ public function getNodeTypes(): array
      */
     public function refactor(Node $node): ?Node
     {
-        if ($this->isObjectType($node->var, Controller::class) && $this->isName($node->name, 'createForm')) {
+        if ($this->isObjectTypes($node->var, [Controller::class, AbstractController::class]) && $this->isName($node->name, 'createForm')) {
```

Comment:
It might be fix for this case, but this can be acually broken when `Symfony\` gets prefixed in `rector.phar`.

Could you update the args so its strings?

```suggestion
        if ($this->isObjectTypes($node->var, ['Symfony\Bundle\FrameworkBundle\Controller\AbstractController', 'Symfony\Bundle\FrameworkBundle\Controller\Controller']) && $this->isName($node->name, 'createForm')) {
```

Reply:
@TomasVotruba I will update.

There are other Symfony imports in this class:

Symfony\Component\Form\FormBuilderInterface
Symfony\Component\Form\FormInterface
Symfony\Component\OptionsResolver\OptionsResolver

Do these also need to be updated?

Reply:
Yes. That would be great!